### DISCUSSION TITLE
关于证书的生成方式的选择

### DISCUSSION THREAD

**Discussion by gzf6** [XTLS/Xray-core#789] [Q&A]
在 [小小白白话文](https://xtls.github.io/Xray-docs-next/document/level-0/ch06-certificates.html#_6-3-%E6%B5%8B%E8%AF%95%E8%AF%81%E4%B9%A6%E7%94%B3%E8%AF%B7)（ECC 证书） 和 [SNI 回落](https://xtls.github.io/Xray-docs-next/document/level-1/fallbacks-with-sni.html#%E7%94%B3%E8%AF%B7-tls-%E8%AF%81%E4%B9%A6)（DNS-01 验证方式） 中证书的生成方式不一样，应该用哪个呢？

**Comment by shenlijun** [comment]
https://www.cyberciti.biz/faq/issue-lets-encrypt-wildcard-certificate-with-acme-sh-and-cloudflare-dns/

这里是dns_cf，可以去acme那里看看各个提供商自己的dns用法

一劳永逸，自动更新不用自己管
